src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -84,8 +85,11 @@
  * session; that is, if the segment is associated with a shared session, it can be accessed by multiple threads;
  * if it is associated with a confined session, it can only be accessed by the thread which owns the memory session.
  * <p>
- * Heap and buffer segments are always associated with a <em>global</em>, shared memory session. This session cannot be closed,
- * and segments associated with it can be considered as <em>always alive</em>.
+ * Heap segments are always associated with the {@linkplain MemorySession#global() global} memory session.
```

Comment:
I've tweaked this text, as I realized the old version was erroneously suggesting that all buffer segments were backed by a global session.